### ISSUE TITLE
Bring in built references as standalone files

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [FullScreenShenanigans/FullScreenMario#314]
```
Running "mocha_phantomjs:all" (mocha_phantomjs) task
ReferenceError: Can't find variable: EightBittr
   at file:///home/travis/build/FullScreenShenanigans/FullScreenMario/Source/References/GameStartr-0.2.0.js:1383
   at file:///home/travis/build/FullScreenShenanigans/FullScreenMario/Source/References/GameStartr-0.2.0.js:1385
Warning: 1 tests failed Use --force to continue.
```

**Comment by JoshuaKGoldberg** [comment]
Looks like this is more of a general problem.

Right now, referenced libraries include all their references in their combined .js files. Instead, they should just include themselves.

This can be done by changing the grunt files to specify the typescript task's dest as the source directory instead of the js file.

**Comment by JoshuaKGoldberg** [comment]
Closed by fe08c3c6f038bbf210089dab672398b3eaaa0c94.
